File: travel/src/main/java/cn/itcast/travel/service/UserService.java
package cn.itcast.travel.service;

import cn.itcast.travel.domain.User;

public interface UserService {
    /**
     * 注册用户
     * @param user
     * @return
     */
    boolean regist(User user);

    /**
     * 激活用户
     * @param code
     * @return
     */
    boolean active(String code);

    /**
     * d登录方法
     * @param user
     * @return
     */
    User login(User user);

    /**
     * 根据激活码判断是否已激活
     * @param code
     * @return
     */
    boolean isActive(String code);
}

File: shiroDemo/src/main/webapp/js/service/userService.js
app.service('userService',function($http){

    this.login=function(entity){
        return $http.post('../user/login.do',entity);
    }
    
    this.findLoginName=function () {
        return $http.get("../user/name.do");
    }

    this.logout=function () {
        return $http.get("../user/logout.do");
    }

});